addons/base_gengo/wizard/base_gengo_translations.py

```diff
@@ -172,42 +146,37 @@ def _update_terms_job(self, cr, uid, job, context=None):
         if job.get('status', False) in ('approved', 'canceled'):
             vals['state'] = 'translated'
         if vals:
-            translation_pool.write(cr, uid, [tid], vals, context=context)
+            IrTranslation.browse(tid).write(vals)
 
-    def _update_terms(self, cr, uid, response, term_ids, context=None):
+    def _update_terms(self, response, terms):
         """
         Update the terms after their translation were requested to Gengo
         """
-        translation_pool = self.pool.get('ir.translation')
-
         vals = {
             'order_id': response.get('order_id', ''),
             'state': 'inprogress'
         }
-
-        translation_pool.write(cr, uid, term_ids, vals, context=context)
+        terms.write(vals)
         jobs = response.get('jobs', [])
         if jobs:
             for t_id, res in jobs.items():
-                self._update_terms_job(cr, uid, res, context=context)
-
+                self._update_terms_job(res)
         return
 
-    def pack_jobs_request(self, cr, uid, term_ids, context=None):
+    def pack_jobs_request(self, terms):
         ''' prepare the terms that will be requested to gengo and returns them in a dictionary with following format
             {'jobs': {
                 'term1.id': {...}
                 'term2.id': {...}
                 }
             }'''
-        base_url = self.pool.get('ir.config_parameter').get_param(cr, uid, 'web.base.url')
-        translation_pool = self.pool.get('ir.translation')
+        BaseUrl = self.env['ir.config_parameter'].get_param('web.base.url')
```

Comment:
should be 'base_url' as it does not hold model registry and same remark for 'User' below.